recipe/meta.yaml

```
@@ -24,36 +23,35 @@ requirements:
     - wheel
     - setuptools
   run:
-    # altair (5.x) will break Streamlit <1.23, see https://github.com/streamlit/streamlit/pull/6219
-    # and https://github.com/streamlit/streamlit/commit/b2e485954538cc18ab4a5542d3410659dafa6b0f
-    - altair >=3.2.0,<5
-    - blinker >=1.0.0
-    - cachetools >=4.0
-    - click >=7.0
-    - importlib-metadata >=1.4
-    - numpy
-    - packaging >=14.1
-    # pandas 2.0 support was added only for Streamlit >=1.22.0, see https://github.com/streamlit/streamlit/commit/0e7b95706623acd2fed4028cf81f5294a387caba
-    - pandas >=0.21.0,<2
-    - pillow >=6.2.0
-    - protobuf >=3.12,<4
+    - altair >=4.0,<6
+    - blinker >=1.0.0,<2
+    - cachetools >=4.0,<6
+    - click >=7.0,<9
+    - importlib-metadata >=1.4,<7
+    - numpy {{ numpy }}
```

```suggestion
    - {{ pin_compatible('numpy') }}
```
Because it's a runtime dependency not it `host`

---

removed pinning completely. {{ pin_compatible('numpy') }} on it's own without numpy in the host does not do anything.